tests/kernels/test_scaled_mm_triton.py

```diff
@@ -0,0 +1,106 @@
+"""Tests for the scaled_mm_triton kernel
+
+Run `pytest tests/kernels/test_scaled_mm_triton.py`.
+"""
+import importlib
+from typing import Optional, Type
+
+import pytest
+import torch
+
+from vllm.platforms import current_platform
+
+device = "cuda"
+
+
+def scaled_mm_torch(a: torch.Tensor,
+                    b: torch.Tensor,
+                    scale_a: torch.Tensor,
+                    scale_b: torch.Tensor,
+                    out_dtype: Type[torch.dtype],
+                    bias: Optional[torch.Tensor] = None) -> torch.Tensor:
+    out = torch.mm(a.to(torch.float32), b.to(torch.float32))
+    out = scale_a * out
+    out = scale_b.T * out
+    out = out.to(out_dtype)
+    if bias is not None:
+        out = out + bias
+
+    return out
+
+
+def get_8bit_types():
+    types = [torch.int8]
+    supports_fp8 = current_platform.has_device_capability(89)
+    if current_platform.is_rocm() and supports_fp8:
```

Comment:
Is this right for rocm though? What does `current_platform.has_device_capability(89)` return for AMD systems?

Reply:
Yes, it returns true on MI300.